index.js

```diff
@@ -289,8 +290,8 @@ function choosePluralForm(text, locale, count) {
 // interpolation placeholder.
 var dollarRegex = /\$/g;
 var dollarBillsYall = '$$';
-var tokenRegex = /%\{(.*?)\}/g;
-function interpolate(phrase, options) {
+function interpolate(phrase, options, tokenPattern) {
+  var tokenRegex = new RegExp(tokenPattern, 'g');
```

Comment:
I think it's useful for developers allowing a customization in the pattern, but now I see your point. 

What do you think about this approach? http://i18next.com/docs/options/#interpolation-options

Would that be acceptable? 

Thanks!
Luciano